### PR TITLE
Minor protolathe removal

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
Removes the bee gun as something research can manufacture, essentially disabling it except as adminspawn.

The beegun is a nonsensical meme weapon designed for LRP servers. It has extremely poor quality spritework and has no purpose on this server, being mechanically essentially useless and only accessible through the already deeply-flawed research system. Furthermore, the bees themselves are buggy, being extremely difficult to kill.

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!review

**Review by Arrow768 — CHANGES_REQUESTED** [review]
If this is indeed the last place where the bee gun is available, then it should be removed completely (code and sprites) and not just patched out from the protolathe

**Comment by TheDanseMacabre** [comment]
> If this is indeed the last place where the bee gun is available, then it should be removed completely (code and sprites) and not just patched out from the protolathe

And so it shall be done

File: code/modules/projectiles/guns/energy/special.dm [hunk 1]
```diff
@@ -96,27 +96,6 @@
 	turret_is_lethal = 0
 	turret_sprite_set = "net"
 
-/obj/item/gun/energy/beegun
-	name = "\improper NanoTrasen Portable Apiary"
-	desc = "An experimental firearm that converts energy into bees, for purely botanical purposes."
-	icon = 'icons/obj/guns/gyrorifle.dmi'
-	icon_state = "gyrorifle"
-	item_state = "gyrorifle"
```
Comment (Arrow768) [inline code/modules/projectiles/guns/energy/special.dm:104]: From what I can see at a quick glance, this icon / dmi file is only used for the beegun.
If all the code referencing it is removed, the icon/dmi should be removed aswell.

**Review by Arrow768 — DISMISSED** [review]

**Review by JohnWildkins — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**